packages/react-components/react-persona/Spec.md

```diff
@@ -1,63 +1,212 @@
+### Text lines
+
+Other than styling and naming, the text lines in Persona remain the same.
 
 ## Sample Code
 
-_Provide some representative example code that uses the proposed API for the component_
+![Avatar](./etc/images/avatar.png)
+
+Persona with Avatar:
+
+```jsx
+<Persona avatar={{ name: 'Kevin Sturgis' }} primaryText="Kevin Sturgis" secondaryText="Software Engineer" />
+```
+
+![PresenceBadge](./etc/images/badge.png)
+
+Persona with PresenceBadge:
+
+```jsx
+<Persona presence={{ status: 'offline', outOfOffice: true }} primaryText="Kevin Sturgis" />
+```
+
+![Avatar and PresenceBadge](./etc/images/avatar_badge.png)
+
+Persona with Avatar + PresenceBadge:
+
+```jsx
+<Persona
+  avatar={{ name: 'Kevin Sturgis' }}
+  presence={{ status: 'offline', outOfOffice: true }}
+  primaryText="Kevin Sturgis"
+  secondaryText="Software Engineer"
+  tertiaryText="Offline"
+/>
+```
 
 ## Variants
 
-_Describe visual or functional variants of this control, if applicable. For example, a slider could have a 2D variant._
+> I will refer to the avatar and presence as content.
+
+There are three alignment variants:
+
+- start: Content on the left and text on the right.
+- center: Content on top and text on the bottom.
+- end: Content on the right and text on the left.
+
+There are 3 content variants:
+
+- Avatar
+- PresenceBadge
+- Avatar + PresenceBadge
+
+There are 2 sizing variants:
+
+- scaled: The content will be resized based on the number of text lines.
+- fixed: The content will remain the same size and the text lines will be styled based on the content's size.
 
 ## API
 
-_List the **Props** and **Slots** proposed for the component. Ideally this would just be a link to the component's `.types.ts` file_
+**Slots**
+
+- `root`: The root slot for Persona.
+- `avatar`: The Avatar to display.
+- `presence`: The PresenceBadge to display.
+- `primaryText`: Primary text.
+- `secondaryText`: Secondary text.
+- `tertiaryText`: Tertiary text.
+- `quaternaryText`: Quaternary text.
+
+**Types**
+
+```ts
+export type PersonaSlots = {
+  root: Slot<'div'>;
+
+  /**
+   * Avatar to display.
+   */
+  avatar?: Slot<typeof Avatar>;
+
+  /**
+   * PresenceBadge to display.
+   */
+  presence?: Slot<typeof PresenceBadge>;
+
+  /**
+   * Primary text to display.
+   */
+  primaryText?: Slot<'span'>;
+
+  /**
+   * Secondary text to display.
+   */
+  secondaryText?: Slot<'span'>;
+
+  /**
+   * Tertiary text to display.
+   */
+  tertiaryText?: Slot<'span'>;
+
+  /**
+   * Quaternary text to display.
+   */
+  quaternaryText?: Slot<'span'>;
+};
+
+/**
+ * Persona Props
+ */
+export type PersonaProps = ComponentProps<PersonaSlots> & {
+  /**
+   * Sizing type to use.
+   *
+   * `fixed`: Text lines adjust to content's size.
+   * `scaled`: Content adjusts its size based on the number of text lines used.
+   */
+  sizing?: 'fixed' | 'scaled';
+
+  /**
+   * Position the Avatar/PresenceBadge will be rendered in.
+   */
+  position?: 'start' | 'center' | 'end';
```

Comment:
How about `textPosition?: 'before' | 'after' | 'below'`. This aligns with other components that use `labelPosition` props.